.env.example

```diff
@@ -32,3 +32,6 @@ ADLDAP_TLS=false
 ADLDAP_SSL=true
 ADLDAP_PORT=389
 
+ADLDAP_LOGIN_FALLBACK=true
```

Comment:
Does this need to be used somewhere?

Reply:
Yeah, it's used [here](https://github.com/UNC-Libraries/jitterbug/blob/70e5348ddb8db4dc1f412ae8304dffe5bd8c762c/config/adldap_auth.php#L69). That was the line where in the previous branch I had changed the second argument to true, not realizing that was actually the default. So the actual value needs to be set in the `.env` file.

Reply:
I guess it doesn't have to be in the example file since it wasn't previously...I could put it in the readme. I thought this way it would be a little clearer.

Reply:
No, this is great--I forgot that it was in the last PR.